### PR TITLE
Task 0

### PR DESCRIPTION
**Pull request by Sora-ar**
I'm adding files from homework 1 by task 0, credited back in the summer, so that I don't get confused about doing the next homework on this repository.

### CODE REVIEW

File: lab1_v3.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        print('long ', longest_length)
+
+
+def zero_search():
+    counter = sum(1 for i in range(N) for j in range(M) if arr[i][j] == 0)
+    print('Number of rows with zero element: ', counter)
+
+
+def print_array(lst):
+    [print(' '.join(map(str, row))) for row in lst]
```
Comment (vloooo) [inline lab1_v3.py]: inappropriate)
let's discuss it later

File: lab1_v4.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+def multiplication_positive_num():
+    for i, row in enumerate(arr):
+        counter = 1
+        for element in row:
+            if element < 0:
+                print(i + 1, ' row: there is a negative element')
+                break
+            counter *= element
+        else:
+            print('Multiplication of positive numbers of ', i + 1, ' row: ', counter)
```
Comment (vloooo) [inline lab1_v4.py]: I think comprehension is possible here

File: lab1_v4.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
+def print_array(lst):
+    [print(' '.join(map(str, row))) for row in lst]
+
+
+def create_array():
+    return [[random.randint(0, 9) for _ in range(M)] for _ in range(N)]
```
Comment (vloooo) [inline lab1_v4.py]: code duplication

File: lab1_v3.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                if count_length > longest_length:
+                    longest_length = count_length
+                    element_long = row[j]
+                    row_longest = i + 1
+            else:
+                count_length = 1
+
+    if longest_length > 1:
+        print(row_longest, ' row that contains the longest series of identical elements: ', element_long)
+        print('long ', longest_length)
```
Comment (vloooo) [inline lab1_v3.py]: f-strings?

File: lab1_v19.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    print('Min sum:', min(min_sum, arr[0][-1]))
+
+
+def sum_non_negativ_rows():
+    for i, row in enumerate(arr):
+        row_sum = sum(row)
+        if all(element >= 0 for element in row):
+            print('Row', i + 1, ': sum =', row_sum)
+        else:
+            print(i + 1, 'row: there is a negative element')
```
Comment (vloooo) [inline lab1_v19.py]: apply ternary operator

File: lab1_v4.py (hunk 2 above)
Comment (Sora-ar) [reply lab1_v4.py]: I can't get a sense of how to do it. Can we review this on the meeting on Tuesday?

**Pull request merged**